wheel_motor_separation: 0.206
wheel_radius: 0.065
publish_tf: true
left_motor_topic: "motor/left"
right_motor_topic: "motor/right"

odom_frame_id: "odom"
base_frame_id: "base_footprint"